### PR TITLE
Wich maker

### PR DESCRIPTION
**Pull request by aefinch**
@abidina

### CODE REVIEW

**Comment by aefinch** [comment]
readme complete, ready for review

**Comment by debragordon** [comment]
There is a lot of commented out code ... I recommend cleaning that out and making the files more readable.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+				<label><input type="checkbox" id="saucebox6" value="barbecue">Barbecue</label><br>
+				<br>
+			</div>
+			<div id="veggieChooser">
+				<label for="veggies"> Veg Out!</label><br>
+				<label><input type="checkbox" id="vegbox1" value="lettuce">Lettuce</label><br>
+				<label><input type="checkbox" id="vegbox2" value="tomato">Tomato</label><br>
+				<label><input type="checkbox" id="vegbox3" value="onion">Onion</label><br>
+				<label><input type="checkbox" id="vegbox4" value="cucumber">Cucumber</label><br>
+				<label><input type="checkbox" id="vegbox5" value="pickle">Pickles</label><br>
```
Comment (debragordon) [inline index.html]: this doesn't match your .js file 

pickle vs pickles ... I think it breaks your total method

File: index.html [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+				<label><input type="checkbox" id="cheesebox4" value="meunster">Meunster</label><br>
+				<label><input type="checkbox" id="cheesebox5" value="mozzarella">Mozzarella</label><br>
+				<br>
+			</div>
+			<div id="sauceChooser">
+				<label for="condiments">Select your sauce(s)!</label><br>
+				<label><input type="checkbox" id="saucebox1" value="mustard">Mustard</label><br>
+				<label><input type="checkbox" id="saucebox2" value="mayo">Mayo</label><br>
+				<label><input type="checkbox" id="saucebox3" value="ranch">Ranch</label><br>
+				<label><input type="checkbox" id="saucebox4" value="oil-vinegar">Oil and Vinegar</label><br>
```
Comment (debragordon) [inline index.html:46]: This doesn't match your .js file oil-vinegar ... you name it oilVinegar in the other file

Comment (GAylabea) [reply index.html:46]: I think there was also a pickle vs. pickles problem.

**Comment by debragordon** [comment]
files are cleaner and the naming conventions are better, good to merge

**Pull request merged**